### PR TITLE
Upgrade to Fedora 28, Elasticsearch 6

### PR DESCRIPTION
**Pull request by yrodiere**
I included an intermediate step to upgrade to ES 5.6.12, in case you want to push a tag for that release too.

### CODE REVIEW

**Comment by yrodiere** [comment]
@Sanne ping?

**Comment by Sanne** [comment]
thanks for the ping, apparently didn't get any notification.

File: Dockerfile [hunk 1]
```diff
@@ -26,7 +26,8 @@ RUN \
 
 COPY log4j2.properties /opt/elasticsearch/config/
 COPY elasticsearch.yml /opt/elasticsearch/config/
-COPY jvm.options /opt/elasticsearch/config/
+COPY jvm.options-override /opt/elasticsearch/config/
+RUN cd /opt/elasticsearch/config/ && cat jvm.options-override >> jvm.options && rm jvm.options-override
```
Comment (Sanne) [inline Dockerfile:30]: This isn't actually working out and I'll revert it. It gets very confusing when you want to customize a property by removing it, or setting a different value for something which is already set.

I prefer being able to see them all in one file.

**Comment by Sanne** [comment]
I merged the component upgrades, but keps the jvm properties in a single file.

Tagging images now..

Thanks!

File: Dockerfile (hunk 1 above)
Comment (yrodiere) [reply Dockerfile:30]: @Sanne As you wish. Did you make sure to use the correct jvm.options file from the new version of Elasticsearch? It changed.

Comment (yrodiere) [reply Dockerfile:30]: Never mind, I can see you did. Sorry for the noise.

Comment (Sanne) [reply Dockerfile:30]: np, good that you watch :+1: 

BTW the problem was I couldn't un-set the JVM GC tuning; with small heaps & having JDK11 those were not making sense.

**Pull request merged**